NinchatSDKSwift/Implementations/Managers/Session Manager/NINChatSessionManager.swift

```diff
@@ -78,6 +78,9 @@ protocol NANChatSessionMessenger {
     
     /** Load channel history. */
     func loadHistory(completion: @escaping CompletionWithError) throws
+
+    /* Close an old session using given credentials async. */
+    func closeSession(credentials: NINSessionCredentials, completion: ((NINResult<Empty>) -> Void)?)
```

Comment:
Isn't the completion unused now?

Reply:
It is used for testing purposes and just simply check if the server returns status code 200 or not.

Reply:
Ok